packages/payment/src/payment.ts

```diff
@@ -41,7 +90,20 @@ export interface Payment {
     customField9?: string;
     customField10?: string;
 
-    paymentMethod?: PaymentMethod;
+    /**
```

Comment:
This is tsdoc syntax, which means this comment is going to be associated as the description for `method`.  Make it a regular comment, not tsdoc, e.g. use `//`.  You can do various things to make it stand out if desired, e.g.
```
// --------------------------
// ...comment goes here....
// --------------------------
```

Reply:
Oh okay, I didn't realize that. I'll update this and any other comments that I added that fit into the same category.